HTseq/featureCount.nf

```diff
@@ -0,0 +1,25 @@
+process featureCount {
+    tag "featureCount"
+    publishDir "$params.outdir/${sample}/counts", mode: 'copy'
+    cpus 4
```

Comment:
Not sure whether we should define resources in the modules. I also like the solution of @sawibo in https://github.com/UMCUGenetics/NextflowModules/blob/dev-useq/IAP-Reloaded/nextflow.config.